src/Common/Dialogs/DeleteDialog.tsx

```diff
@@ -13,6 +30,18 @@ export const DeleteDialog: React.FC<DeleteDialogProps> & { Description?: React.F
         props.delete()
     }
 
+    const handleKeyDown = async (event: KeyboardEvent) => {
+        if (event.key === 'Enter' && !isDeleteDisabled) {
+            event.preventDefault()
+
+            if (props.shouldStopPropagation) {
```

Comment:
Should we move this outside parent if or is it required for something else?

Reply:
Required only on enter